### PR TITLE
Add issues related to auditability and tracking risks.

### PR DESCRIPTION
**Pull request by msporny**

### CODE REVIEW

**Review by rhiaro — APPROVED** [review]

File: index.bs [hunk 1]
```diff
@@ -177,12 +177,20 @@ The following steps must be executed to resolve the DID document from a Web DID:
...
+
 ### Update
 
 To update the DID document, the `did.json`has to be updated. Please note that
 the DID will remain the same, but the contents of the DID document could change,
 e.g., by including a new verification key or adding service endpoints.
 
+Issue: There is no authentication or authorization mechanism applied to the DID Document, leaving it unprotected from modification by an attacker.
+
+Issue: There is no HTTP API specified for the update process leaving programmatic registrations and management to be defined by a proprietary process.
```
Comment (rhiaro) [inline index.bs]: Raised: https://github.com/w3c-ccg/did-method-web/issues/11

File: index.bs [hunk 2]
```diff
@@ -177,12 +177,20 @@ The following steps must be executed to resolve the DID document from a Web DID:
...
+
+Issue: Reads enable pervasive tracking of DID use across the Internet. There is currently no mitigation for this privacy-violating mechanism.
+
 ### Update
 
 To update the DID document, the `did.json`has to be updated. Please note that
 the DID will remain the same, but the contents of the DID document could change,
 e.g., by including a new verification key or adding service endpoints.
 
+Issue: There is no authentication or authorization mechanism applied to the DID Document, leaving it unprotected from modification by an attacker.
```
Comment (rhiaro) [inline index.bs]: Raised https://github.com/w3c-ccg/did-method-web/issues/12

File: index.bs [hunk 3]
```diff
@@ -177,12 +177,20 @@ The following steps must be executed to resolve the DID document from a Web DID:
     requirements as described in
     [Security Considerations](Security-Considerations).
 
+Issue: The read mechanism does not provide any sort of auditability on the DID, which leaves this DID Method open to insider-threat attacks, among others.
```
Comment (rhiaro) [inline index.bs:180]: Raised https://github.com/w3c-ccg/did-method-web/issues/13

File: index.bs [hunk 4]
```diff
@@ -177,12 +177,20 @@ The following steps must be executed to resolve the DID document from a Web DID:
...
+
+Issue: Reads enable pervasive tracking of DID use across the Internet. There is currently no mitigation for this privacy-violating mechanism.
```
Comment (rhiaro) [inline index.bs:182]: Raised https://github.com/w3c-ccg/did-method-web/issues/14

**Review by rhiaro — APPROVED** [review]

File: index.bs (hunk 1 above)
Comment (dmitrizagidulin) [reply index.bs]: This is intentional, aside from http GET for reads, there are no universal http update methods that work for all domains. (This isn't Solid, yo :) )

File: index.bs (hunk 2 above)
Comment (dmitrizagidulin) [reply index.bs]: This one doesn't quite make sense, or is a duplicate of issue #13.

File: index.bs (hunk 1 above)
Comment (OR13) [reply index.bs]: IMO this could be considered a "security through diversity" property.

File: index.bs (hunk 2 above)
Comment (OR13) [reply index.bs]: Another "bug/feature" in terms of usability.

**Review by OR13 — APPROVED** [review]

**Comment by dmitrizagidulin** [comment]
@msporny - I am +1 on this PR, provided the `There is no authentication or authorization mechanism applied to the DID Document` issue is removed. (See my comments on corresponding issue #12.)

**Review by dmitrizagidulin — CHANGES_REQUESTED** [review]
Also, please remove the `There is no HTTP API specified for the update process` issue. This is intentional, and core to the design.

File: index.bs (hunk 2 above)
Comment (rhiaro) [reply index.bs]:
```suggestion
Issue: This DID method does not specify any authentication or authorization mechanism for writing to the DID Document, leaving it up to implementations to protect from modification by an attacker.
```

File: index.bs (hunk 1 above)
Comment (rhiaro) [reply index.bs]:
```suggestion
Note: There is no HTTP API specified for the update process leaving programmatic registrations and management to be defined by each implementation.
```

Okay I'm almost convinced this is a feature, but do think we should expand on it more in the future.

File: index.bs (hunk 2 above)
Comment (dmitrizagidulin) [reply index.bs]: @rhiaro - +1, thanks!

File: index.bs (hunk 1 above)
Comment (dmitrizagidulin) [reply index.bs]: @rhiaro - better, thanks! Though we should change 'by a proprietary process' -> 'to be defined by each individual domain's implementation'.

**Review by dmitrizagidulin — APPROVED** [review]
👍 
Thanks again, @msporny and @rhiaro.

**Review by OR13 — APPROVED** [review]

**Pull request merged**